### PR TITLE
Grids

### PR DESCRIPTION
**Pull request by AshutoshM29**
I have added "Grid Layouts" in the CL, please provide feedback :)

Preview Link: https://deploy-preview-19--cleanui-grid.netlify.app/pages/grids/grids

https://user-images.githubusercontent.com/89376505/159640873-d671ec34-0649-411a-882c-4a88a4660eb6.mp4

### CODE REVIEW

**Review by thenorthkun — APPROVED** [review]
UI Suggestion : you can give `width : 100%` to the img tag for better visibility.
Everything else is good.

File: Pages/Grids/grids.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                <div class="div-txt">
+                    This is an example of 50-50 Grid. Here we have an Image and some Text !!
+                </div>
+            </div>
+        </section>
+
+        <p class="body-r">Copy the given code to create a 50-50 Grid</p>
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-50-50%2522%253E%250A%2509%253Cdiv%253E%250A%2509%2509%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F500%252F500%2522%2520alt%253D%2522responsive-img%2522%2520%252F%253E%250A%2509%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522div-txt%2522%253E%250A%2520%2520%2520%2520%2509This%2520is%2520an%2520example%2520of%252050-50%2520Grid.%2520Here%2520we%2520have%2520an%2520Image%2520and%2520some%2520Text%21%21%250A%2509%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  style="width: 80rem; height: 40rem; border:0; transform: scale(1); overflow:hidden;"
```
Comment (rajaranjanmr) [inline Pages/Grids/grids.html:71]: suggestion : try to avoid inline css

File: Pages/Grids/grids.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                <div class="div-txt">
+                    This is an example of 60-40 Grid. Here we have an Image and some Text !!
+                </div>
+            </div>
+        </section>
+
+        <p class="body-r">Copy the given code to create a 60-40 Grid</p>
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-60-40%2522%253E%250A%2509%253Cdiv%253E%250A%2509%2509%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F600%252F300%2522%2520alt%253D%2522responsive-img%2522%2520%252F%253E%250A%2509%253C%252Fdiv%253E%250A%2509%253Cdiv%2520class%253D%2522div-txt%2522%253E%250A%2509%2509This%2520is%2520an%2520example%2520of%252060-40%2520Grid.%2520Here%2520we%2520have%2520an%2520Image%2520and%2520some%2520Text%21%21%250A%2509%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  style="width: 80rem; height: 40rem; border:0; transform: scale(1); overflow:hidden;"
```
Comment (rajaranjanmr) [inline Pages/Grids/grids.html:90]: same here

**Review by rajaranjanmr — APPROVED** [review]
rest seems to be good and can be merged

File: Pages/Grids/grids.html (hunk 1 above)
Comment (AshutoshM29) [reply Pages/Grids/grids.html:71]: > suggestion : try to avoid inline css

I didn't do this styling, it's imported from carbon

**Pull request merged**